source/documentation/standards/hosting.md.erb

```diff
@@ -0,0 +1,66 @@
+---
+title: How to host services
+last_reviewed_on: 2020-09-01
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+At MOJ you should use the [Cloud
+Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) to
+host your service unless you're advised otherwise by a member of [the
+hosting team](https://peoplefinder.service.gov.uk/teams/hosting). Your
+team can get started on the Cloud Platform without letting us know;
+just follow [the Cloud Platform documentation
+](https://user-guide.cloud-platform.service.justice.gov.uk/) and get in
+touch with the team if you need help.
+
+If you think that your service might require other hosting, [get in
+touch with the hosting
+team](mailto:hosting-leads@digital.justice.gov.uk) and we'll help
+identify which of our other hosting options is best for your service,
+or set up extra accounts if we've already identified the right
+environment for you:
+
+- AWS Modernisation Platform (currently in alpha)
```

Comment:
“alpha” might be a bit optimistic… But maybe we can use this as a call to action? @jakemulley waddya think?

Reply:
I think alpha is too optimistic & agree with the CTA. Would be good to direct people to talk to us.

Maybe this list should be ordered on basis of preference if it's not already. I'm unsure if self-managed is something we'd want to push?

Reply:
I've added a bit more context on Modernisation Platform, and re-ordered the list a little. I don't want to _push_ self-managed, you're right, but I think it should at least be listed…